pkg/activator/revision.go

```diff
@@ -143,17 +141,17 @@ func (r *revisionActivator) getRevisionEndpoint(revision *v1alpha1.Revision) (en
 func (r *revisionActivator) ActiveEndpoint(namespace, name string) ActivationResult {
```

Comment:
Comment for public function

Reply:
Ditto as above, but at least this does something, so I added a few words.